app/src/main/kotlin/com/saltedge/authenticator/features/connections/list/ConnectionItemHolder.kt

```diff
@@ -48,7 +48,8 @@ class ConnectionItemHolder(parent: ViewGroup, private val listener: ListItemClic
     fun bind(item: ConnectionViewModel) {
         logoImageView.loadRoundedImage(
             imageUrl = item.logoUrl,
-            placeholderId = R.drawable.ic_logo_bank_placeholder
+            placeholderId = R.drawable.ic_logo_bank_placeholder,
```

Comment:
by the new design we should use gray shape as placeholder, not ic_logo_bank_placeholder

Reply:
discussed it with @AlexLobaciov